src/test/java/ua/com/juja/microservices/keepers/slackbot/dao/impl/RestKeeperRepositoryTest.java

```diff
@@ -93,4 +94,20 @@ public void shouldThrowExceptionWhenSendAddKeeperToKeepersServiceThrowException(
         //when
         keeperRepository.addKeeper(new KeeperRequest("qwer", "67ui", "teems"));
     }
+
+    @Test
+    public void shouldReturnKeeperDirections() {
+        //given
+        String expectedRequestHeader = "application/json";
+        mockServer.expect(requestTo(urlBaseKeeper + "/0000-1111"))
+                .andExpect(method(HttpMethod.GET))
+                .andExpect(request -> assertThat(request.getHeaders().getContentType().toString(), containsString(expectedRequestHeader)))
+                .andRespond(withSuccess("[\"direction1\",\"direction2\"]", MediaType.APPLICATION_JSON));
+        //when
+        List<String> actualList = keeperRepository.getKeeperDirections("0000-1111");
+        // then
+        mockServer.verify();
+        assertEquals(actualList.size(), 2);
```

Comment:
I mean that not need check  assertEquals(actualList.size(), 2), because on the next row you check that in array have only two certain elements.